pom.xml

```diff
@@ -2,94 +2,81 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0"
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-	<description>An Elasticsearch plugin for network address analysis</description>
+    <description>Elasticsearch plugin for network address and path analysis</description>
     <modelVersion>4.0.0</modelVersion>
-    <groupId>com.ofir123</groupId>
-    <artifactId>elasticsearch-network-analysis</artifactId>
-    <version>5.3</version>
+    <groupId>com.matan129</groupId>
```

Comment:
Maybe a new groupId?
(or I can leave it on your name, it's fine by me)

Reply:
`com.elasticsearch.networkanalysis`?

Reply:
No, this is too confusing with official ES repos (the ES package are `org.elasticsearch` something).
nvm, let's leave it this way